conftest.py

```diff
@@ -11,6 +11,7 @@
     'pytest_plugins.marker_deselection',
     'pytest_plugins.markers',
     'pytest_plugins.metadata_markers',
+    'pytest_plugins.skip_component_tests',
```

Comment:
The plugin is misnamed, its not skipping tests, its deselecting them based on marker - which I would expect to see in the `marker_deselection` module.

Reply:
Agree, that the plugin is misnamed, TBH I forgot to rename the plugin because initially, I thought to skip the tests and later I thought to uncollect them hence it was left unchanged.

I thought to use the `marker_deselection` plugin but that's more in general created for infra dependent marker deselection and I hope that plugin will be removed when we will have the required infra in place!

Reply:
I don't think the infra related markers, or the module implementing their behavior, will get removed. Even when we internally have provisioning systems setup, we will want to continue controlling test collection.

This could also go into the meta filtering module that implements the `--component` CLI option and controls collection based on the component mark.

This is definitely a useful utility outside the scope of virtwho, and definitely shouldn't be hardcoded for any components.

Reply:
@mshriver Moved the code to `marker_deselection` plugin as per discussion in yesterdays JFT meeting.